packages/frontend/src/components/common/Table/Table.styles.ts

```diff
@@ -106,8 +107,15 @@ export const TableFooter = styled.div`
     margin-top: 10px;
 `;
 
+const FontSyles = `
+    font-size: 13px;
```

Comment:
I did this this way so that I could put these directly on the cells and header so they would be more specific than the Blueprint styles.